### ISSUE TITLE
iOS does not support all of the standard user attributes that Android does

### ISSUE THREAD

**Issue by Jordan-Nelson** [aws-amplify/amplify-flutter#598]
**Describe the bug**
Android support more standard user attributes than iOS. Android supports the standard attributes "profile", "updated_at", "website", and "zoneinfo". These attributes are not supported on iOS. If a customer attempts to add these on iOS, amplify-flutter will attempt to add them as custom attributes.

**To Reproduce**
1. Attempt to add create a new user with one of these attributes OR update the attribute on an existing user

**Example w/ Sign Up**
```dart
var userAttributes = {
  // works on Android, but not iOS
  'website': 'www.example.com',
};
var res = await Amplify.Auth.signUp(
    username: 'testuser001',
    password: 'password123',
    options: CognitoSignUpOptions(userAttributes: userAttributes));
```
**Example w/ updateAttribute**
```dart
await Amplify.Auth.updateUserAttribute(
  // works on Android, but not iOS
  userAttributeKey: 'website',
  value: 'www.example.com',
);
```

**Expected behavior**
These four attributes should be supported when targeting iOS as well.

**Platform**
Amplify Flutter current supports iOS and Android.  This issue is reproducible in (check all that apply):
[] Android
[x] iOS

**Additional context**
amplify-ios currently doesn't support these attributes. There may be a workaround that could be implemented in amplify-flutter, which is to add these four attributes using `.unknown()` instead of `.custom()`.

**Comment by Jordan-Nelson** [comment]
Issue opened currently in amplify-android for this: https://github.com/aws-amplify/amplify-android/issues/787

**Comment by lawmicha** [comment]
if amplify flutter adds them as custom, that will actually end up prepending with "custom:" prefix which would mean an app on Android will send "profile", "zoneinfo", etc. and an app on iOS will send "custom:profile", "custom:zoneinfo"

**Comment by lawmicha** [comment]
another scenario is, what happens if i use amplify flutter and create a .custom attribute? is this possible? If so, that would mean iOS sends it as "custom:[custom_value]" and Android sends it as "[custom_value]"

**Comment by lawmicha** [comment]
> amplify-ios currently doesn't support these attributes. There may be a workaround that could be implemented in amplify-flutter, which is to add these four attributes using .unknown() instead of .custom().

oh i forgot about this, so iOS has `unknown` and `custom` to support both scenarios. I think iOS should update it's enum case to include the extra fields that Android has to be in parity

**Comment by Jordan-Nelson** [comment]
> another scenario is, what happens if i use amplify flutter and create a .custom attribute? is this possible? If so, that would mean iOS sends it as "custom:[custom_value]" and Android sends it as "[custom_value]"

This is an issue as well (tracked [here](https://github.com/aws-amplify/amplify-flutter/issues/596)). Amplify-android requires `"custom:"` be included in the attribute key, but amplify-ios requires that it is not included. There is a partial fix in amplify-flutter for this, but there is still an inconsistency between the two platforms when using amplify-flutter. The issue has more info.

**Comment by Jordan-Nelson** [comment]
This has been fixed in version 0.1.6. "profile", "updated_at", "website", and "zoneinfo" are now supported on both platforms.
